websocket/2.1/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Jakarta WebSocket 2.1 (under development)"
+date: 2021-04-15
+summary: "Release for Jakarta EE 10"
+---
+Jakarta WebSocket defines a API for Server and Client Endpoints
+for the WebSocket protocol (RFC6455).
+
+* [Jakarta WebSocket 2.1 Release Record](https://projects.eclipse.org/projects/ee4j.websocket/releases/2.1)
```

Comment:
@markt-asf   
Could you delete the lines from the Release Record link to the maven artifact link in '_index.md' ?  
These lines would be added for Progress- and/or Release Reviews.

Reply:
Done.